yt_dlp/extractor/infoq.py

```diff
@@ -114,14 +111,10 @@ def _real_extract(self, url):
         video_title = self._html_extract_title(webpage)
         video_description = self._html_search_meta('description', webpage, 'description')
 
-        if '/cn/' in url:
-            # for China videos, HTTP video URL exists but always fails with 403
-            formats = self._extract_bokecc_formats(webpage, video_id)
```

Comment:
Are you sure this is no longer present (even from china)? If not, leave the code there

Reply:
Possibly from China it's available so I will leave it. The test case fails though so there will be no test for that part of the code, I'll leave the test case too but skip it.